File: src/main/scala/com/bombproj/vo/ProjectDataTableVO.java
package com.bombproj.vo;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

import java.util.Date;

@Data
public class ProjectDataTableVO {

    private String id;
    private String tableName;
    private String tableDesc;
    private String type;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm")
    private Date updateTime;
    private String createUser;

}

File: src/main/scala/com/bombproj/vo/ProjectDataTableVO.scala
package com.bombproj.vo

import java.util.Date

import com.fasterxml.jackson.annotation.JsonFormat

import scala.beans.BeanProperty

case class ProjectDataTableVO() {

    @BeanProperty var id: String = _
    @BeanProperty var tableName: String = _
    @BeanProperty var tableDesc: String = _
    @BeanProperty var `type`: String = _
    @BeanProperty
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm") var updateTime: Date = _
    @BeanProperty var createUser: String = _

}
